.github/workflows/build.yml

```diff
@@ -349,7 +349,6 @@ jobs:
       - name: Set matrix environments (Push to master)
         if: github.event_name == 'push' && github.ref == 'refs/heads/master'
         run: |
-            echo "MATRIX_ENVIRONMENTS={\"environment\":[\"Production\"]}" >> $GITHUB_ENV
```

Comment:
This variable is still used in the workflow. Either set it to empty list or delete the other deployment job.